src/providers/google/gdrive-document.ts

```diff
@@ -0,0 +1,207 @@
+import BaseSyncHandler from "../BaseSyncHandler";
+import CONFIG from "../../config";
+import { SyncProviderLogEvent, SyncProviderLogLevel } from '../../interfaces'
+import {
+    SyncResponse,
+    SyncHandlerPosition,
+    SyncHandlerStatus,
+} from "../../interfaces";
+import { SchemaDocument } from "../../schemas";
+import { google, drive_v3 } from "googleapis";
+import { OAuth2Client } from 'google-auth-library';
+import { GaxiosResponse } from "gaxios";
+import { GoogleDriveHelpers } from "./helpers";
+
+const _ = require("lodash");
+
+export default class GoogleDriveDocument extends BaseSyncHandler {
+
+    public getName(): string {
+        return "google-drive-documents";
+    }
+
+    public getSchemaUri(): string {
+        return CONFIG.verida.schemas.DOCUMENT;
+    }
+
+    public getProviderApplicationUrl(): string {
+        return "https://drive.google.com";
+    }
+    
+    public getGoogleAuth(): OAuth2Client {
```

Comment:
Inherit from GoogleHandler and use parent `getGoogleAuth()` method.

Reply:
> Inherit from GoogleHandler and use parent `getGoogleAuth()` method.

Solved